src/pybind/mgr/dashboard/services/sso.py

```diff
@@ -203,7 +203,7 @@ def handle_sso_command(cmd):
                 f = open(idp_metadata, 'r')
                 idp_metadata = f.read()
                 f.close()
-            except FileNotFoundError:
+            except (FileNotFoundError, OSError):
```

Comment:
Is it somehow possible to filter the ``OSError`` by ``errno``. I have the feeling catching ``OSError`` here is to broad.

Reply:
@votdev 
Errno contains ENOTSUP, which is again rather vague/broad.